rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/statements.kt

```diff
@@ -106,8 +92,63 @@ internal fun RpgParser.SelectstatementContext.toAst(conf: ToAstConfiguration = T
         }
         other = SelectOtherClause(statementsOfLastWhen.subList(indexOfOther + 1, statementsOfLastWhen.size), position = otherPosition)
     }
+    val result = beginselect().csSELECT().cspec_fixed_standard_parts().result.text
+    val position = toPosition(conf.considerPosition)
+    val dataDefinition = beginselect().csSELECT().cspec_fixed_standard_parts().toDataDefinition(result, position, conf)
+    return SelectStmt(
+        cases = whenClauses,
+        other = other,
+        dataDefinition = dataDefinition,
+        position = toPosition(conf.considerPosition)
+    )
+}
+
+internal fun RpgParser.BegindoContext.toAst(
```

Comment:
Why did you import `com.smeup.rpgparser.RpgParser.BlockContext` and in this case you didn't import `RpgParser.BegindoContext` ?

Reply:
Should you ask it to intellij... In general I don't care